src/main/java/frc/robot/utils/Pathweaver.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.utils;
+
+import org.montclairrobotics.sprocket.SprocketRobot;
+import org.montclairrobotics.sprocket.drive.DTStep;
+import org.montclairrobotics.sprocket.drive.DTTarget;
+import org.montclairrobotics.sprocket.drive.DriveTrain;
+import org.montclairrobotics.sprocket.geometry.Radians;
+import org.montclairrobotics.sprocket.geometry.Vector;
+import org.montclairrobotics.sprocket.geometry.XY;
+import org.montclairrobotics.sprocket.motors.*;
+import org.montclairrobotics.sprocket.motors.Module;
+import org.montclairrobotics.sprocket.utils.PID;
+import org.montclairrobotics.sprocket.utils.Togglable;
+
+import edu.wpi.first.wpilibj.Notifier;
+
+import jaci.pathfinder.Pathfinder;
+import jaci.pathfinder.PathfinderFRC;
+import jaci.pathfinder.Trajectory;
+import jaci.pathfinder.followers.EncoderFollower;
+
+public class Pathweaver implements DTStep, Togglable{
+
+    Trajectory trajectory;
+    boolean enabled;
```

Comment:
How does this get enabled? 
Should be something from the dashboard? 
Or on the joystick

Reply:
@GarrettBurroughs i would have it from dashboard